.bintray.json

```diff
@@ -0,0 +1,35 @@
+{
+    /* Bintray package information.
+       In case the package already exists on Bintray, only the name, repo and subject
+       fields are mandatory. */
```

Comment:
I suppose so since it parses without any errors... the comments here were from an example.